lib/modules/manager/bazel-module/extract.ts

```diff
@@ -0,0 +1,20 @@
+import { logger } from '../../../logger';
+import { LooseArray } from '../../../util/schema-utils';
+import type { PackageFileContent } from '../types';
+import { ToBazelDep } from './bazel-dep';
+import { parse } from './parser';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFileContent | null {
+  try {
+    const records = parse(content);
+    return LooseArray(ToBazelDep)
+      .transform((deps) => (deps.length ? { deps } : null))
+      .parse(records);
+  } catch (err) {
+    logger.error({ err }, 'BazelModule');
```

Comment:
```suggestion
    logger.debug({ err, fileName }, 'Failed to parse bazel module file');
```
we usually only debug log this, as it's not a fatal error